src/main/java/jalasoft/com/utils/TaskBuilder.java

```diff
@@ -0,0 +1,32 @@
+package jalasoft.com.utils;
+
+import javax.servlet.http.HttpServletRequest;
+
+import jalasoft.com.models.CustomDate;
+import jalasoft.com.models.Status;
+import jalasoft.com.models.Task;
+
+public class TaskBuilder {

```

Comment:
Do you are trying to apply the builder pattern?

Reply:
No, it is just two methods that build a Task or Date object given the http request from any jsp file. So, I am changing the name of this class to Utils and making the methods static.